### PR TITLE
Move invite codes to be server-side

### PR DESCRIPTION
**Pull request by nalinbhardwaj**
On local dev, before entering the full string "testnet" it will say "Offline?" since we don't run an API server with chainID 8453 but I think that's fine since it only affects us internally.

Also fix capitalisation bug in key rotation push notifier.

### CODE REVIEW

File: packages/daimo-api/src/pushNotifier.ts [hunk 1]
```diff
@@ -280,7 +280,7 @@ export class PushNotifier {
   getPushMessagesFromKeyRotations(logs: SigningKeyAddedOrRemovedLog[]) {
     const messages: ExpoPushMessage[] = [];
     for (const log of logs) {
-      const addr = log.address;
+      const addr = getAddress(log.address);
```
Comment (dcposch) [inline packages/daimo-api/src/pushNotifier.ts:283]: 👍

File: packages/daimo-api/src/env.ts [hunk 2]
```diff
@@ -263,3 +263,6 @@ export type ReadOnlyContractType<TAbi extends Abi> = GetContractReturnType<
   TAbi,
   PublicClient<Transport, Chain>
 >;
+
+export const DAIMO_INVITE_CODES =
+  process.env.DAIMO_INVITE_CODES || "zuconnect,devconnect";
```
Comment (dcposch) [inline packages/daimo-api/src/env.ts:268]: nit but i'd do something like

```ts
export const daimoInviteCodes = new Set(
  (process.env.DAIMO_INVITE_CODES || "zuconnect,devconnect").split(',')
);
```

File: packages/daimo-api/test/pushNotifier.test.ts [hunk 3]
```diff
@@ -14,9 +14,9 @@ import { NoteOpLog } from "../src/contract/noteIndexer";
 import { OpIndexer } from "../src/contract/opIndexer";
 import { PushNotifier } from "../src/pushNotifier";
 
-const addrAlice = "0x061b0a794945fe0Ff4b764bfB926317f3cFc8b94";
-const addrBob = "0x061b0a794945fe0Ff4b764bfB926317f3cFc8b93";
-const addrCharlie = "0x061b0a794945fe0Ff4b764bfB926317f3cFc8b95";
+const addrAlice = getAddress("0x061b0a794945fe0Ff4b764bfB926317f3cFc8b94");
+const addrBob = getAddress("0x061b0a794945fe0Ff4b764bfB926317f3cFc8b93");
+const addrCharlie = getAddress("0x061b0a794945fe0Ff4b764bfB926317f3cFc8b95");
```
Comment (dcposch) [inline packages/daimo-api/test/pushNotifier.test.ts:19]: 👍

**Review by dcposch — APPROVED** [review]
LGTM

File: packages/daimo-api/src/env.ts (hunk 2 above)
Comment (nalinbhardwaj) [reply packages/daimo-api/src/env.ts]: done

**Pull request merged**